software/wygiwyh.yml

```
@@ -0,0 +1,11 @@
+name: "WYGIWYH"
+website_url: "https://github.com/eitchtee/WYGIWYH"
+source_code_url: "https://github.com/eitchtee/WYGIWYH"
+description: "Simple and powerful finance tracker"
```

`ERROR:awesome_lint.py: WYGIWYH: description does not end with a dot`

```suggestion
description: "Simple and powerful finance tracker."
```